Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt

```diff
@@ -173,17 +183,33 @@ private fun Day(name: String) {
 @Composable
 private fun DayContainer(
     modifier: Modifier = Modifier,
+    selected: Boolean = false,
     onClick: () -> Unit = { },
     onClickEnabled: Boolean = true,
     backgroundColor: Color = Color.Transparent,
+    onClickLabel: String? = null,
     content: @Composable () -> Unit
 ) {
     // What if this doesn't fit the screen? - LayoutFlexible(1f) + LayoutAspectRatio(1f)
+    val stateDescriptionLabel = stringResource(
+        if (selected) R.string.state_descr_selected else R.string.state_descr_not_selected
+    )
```

Comment:
@JoseAlcerreca do you know of a prettier way to get a string resource inside a semantics modifier? I don't like extracting it here into a val, but I don't know of a better way right now.

Reply:
I don't think there's a better way, we do that already
https://github.com/android/compose-samples/blob/fd91e3af7efd75afe8153c043b3c666e3be41036/Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt#L66